helm/hpcc/values.yaml

```diff
@@ -122,6 +135,13 @@ eclagent:
   useChildProcesses: false
   ## type may be 'hthor' (the default) or 'roxie', to specify that the roxie engine rather than the hthor engine should be used for eclagent workunit processing
   type: hthor
+  #childContainersResources:
```

Comment:
would "childContainerResources" be clearer ?

Reply:
How about workerContainerResources?